server/src/main/java/com/lunchtime/controllers/FeedbackController.java

```diff
@@ -0,0 +1,29 @@
+package com.lunchtime.controllers;
+
+import java.util.List;
+import com.lunchtime.models.Feedback;
+import com.lunchtime.service.FeedbackService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+
+@RestController
+public class FeedbackController {
+
+  private final FeedbackService feedbackService;
+
+  public FeedbackController(FeedbackService feedbackService) {
+    this.feedbackService = feedbackService;
+  }
+
+
+  @GetMapping("/api/feedback/{id}")
+  public ResponseEntity<List<Feedback>> getById(@PathVariable("id") Long id) {
+
+    List<Feedback> page = feedbackService.findByRestIdOrderByDateDesc(id);
```

Comment:
Name `getById(id)` can lead to misunderstanding. As it is Feedback endpoint api user can send feedbackId, not restaurantId
You can use endpoint GET http://www.example.com/restaurants/12345/feedback or something else. As you need to get all feedback in some specific restaurant
https://www.restapitutorial.com/lessons/httpmethods.html